sunpy/lightcurve/sources/erne.py

```diff
@@ -0,0 +1,74 @@
+from datetime import timedelta,datetime
+
+from astropy.io import ascii
+from astropy.table import Table, Column
+
+from sunpy.time import TimeRange
+
+def _parse_txt(filepath):
+    """
+    Parses a SOHO/ERNE file from
+    http://srl.utu.fi/erne_data/carrot/carrota.html
+    http://srl.utu.fi/erne_data/carrot/carrotp.html
+
+    """
+    
+    #Reading in Data along with header
+    data = ascii.read(filepath, delimiter = "\s", data_start = 2 ) 
+
+    header = data.colnames
+    for i in range(len(header)):
+        header[i] = data[0][header[i]]
```

Comment:
I don't like this either...  - though I don't understand what's going on anyway... I think you could do:
```python
header = [data[0][key] for key in data.colnames]
```